lib/index.js

```diff
@@ -68,22 +75,50 @@ exports.default = function (sails) {
   };
 
   // setup outside like this to allow use of the compiler in http.customMiddleware
-  hook.compiler = (0, _webpack2.default)(config.hook.config, function (err, stats) {
-    if (err) throw err;
-    sails.log.info('sails-hook-webpack: Webpack loaded.');
-    sails.log.silly('sails-hook-webpack: ', stats.toString());
-    if (process.env.NODE_ENV === 'development') {
-      sails.after('lifted', () => {
-        setTimeout(function () {
-          sails.log.info('sails-hook-webpack: Watching for changes...');
-          hook.compiler.watch(config.hook.watchOptions, hook.afterBuild.bind(hook));
-        }, config.watchDelay);
+  if (process.env.NODE_ENV === 'development') {
+      const thread = spawn(function() {});
+      thread.run(function setUpWatch(input, done) {
```

Comment:
instead of input, sails can be sent as a context, and webpack config can be extracted from sails namespace.

Reply:
The object cannot be passed as input to the function, if passed, it is taken as JSON. Hence, no methods of sails can be executed.